src/mongodb/mongo.ts

```diff
@@ -4,7 +4,13 @@ import { LambdaLogger } from "../logger";
 
 const logger = new LambdaLogger("ee-utils/mongodb", logReferences);
 
-export const connectToDatabaseViaEnvVar = async (): Promise<void> => {
+export const connectToDatabaseViaEnvVar = async (
+  options = {
+    ssl: true,
+    tlsCAFile: `${__dirname}/global-bundle.pem`,
+    retryWrites: false,
```

Comment:
Is there a reason we're switching retries off?  I can see this is inherited from the older code but I don't know why we want to disable an error correction mechanism here.

Reply:
I don't actually know, I can't find it as an option here https://mongoosejs.com/docs/connections.html#options

Reply:
Yeah, as far as I could tell by looking at it it's either ignored or it's [this](https://www.mongodb.com/docs/manual/core/retryable-writes).  But even then I'm having a really hard time understanding what it actually *does*.

Reply:
I've decided to just remove this now